spec/lib/atlas/lexer_spec.rb

```diff
@@ -121,27 +121,35 @@
           is_expected.to match_atlas_tokens(Atlas::ValueToken.new('shipping_address'), Atlas::ExistsToken)
         end
       end
+
+      describe 'NOTEXISTS' do
+        let(:input) { 'shipping_address NOTEXISTS' }
+
+        it do
+          is_expected.to match_atlas_tokens(Atlas::ValueToken.new('shipping_address'), Atlas::NotExistsToken)
+        end
+      end
     end
 
     context 'operator composition' do
       let(:input) do
-        %(sparkAssigned NOTEQ 2018-12-01 AND
```

Comment:
This seems a bit unrelated to the PR main goal ... is the intent to make the spec more generic?

Reply:
Yes, I'm not sure if we even need this context tbh